R/round.R

```diff
@@ -1,6 +1,6 @@
-#' Round or truncate to a multiple of seconds
+#' Round a hms object
```

Comment:
Pronounced: [en aitsch-emm-ess] or something like that.

```suggestion
#' Round an hms object
```